docs/use-cases/nft-auction/index.md

```diff
@@ -0,0 +1,28 @@
+# NFT Auction
+
+> Layer 1 redeemable vouchers negotiated on layer 2.
+
+NFT drops can be a real problem when run on L1, generating a lot of UTxO and transactions in a short amount of time, clogging the network and leading to congestion and general slowdowns. The problem seems to come not much from the NFTs themselves but from the large of number of bidding transactions people post to grab some.
+
+We sketch a way to run NFT auctions inside Hydra Head that would alleviate that problem. Here is an outline of the sequence of transactions involved, both on-chain and off-chain:
+
+![](./diagram.png)
+
+- The auctioneer forges some "participation tokens" (a.k.a PT) representing participation in an auction for some NFT;
```

Comment:
I would call them "voucher tokens  (VT)" to not mix them with actual Hydra PTs and also refer back to your tagline (line 3)